media/css/m24/components/navigation-refresh.scss

```diff
@@ -528,22 +528,22 @@
     }
 }
 
-// page content
+// page content - hide sub nav and page content when nav menu is open
 // This is not part of Protocol, it is specifc to Bedrock; should be documented in future
 body:has(.c-navigation-items.mzp-is-open) {
-    #outer-wrapper {
+    .c-sub-navigation {
         display: none;
 
         @media #{$mq-md} {
             display: block;
         }
     }
-}
 
-#outer-wrapper {
-    margin-top: 48px;
+    #outer-wrapper {
+        display: none;
 
-    @media #{$mq-md} {
-        margin-top: 0;
+        @media #{$mq-md} {
+            display: block;
+        }
     }
 }
```

Comment:
**Suggestion:** it occurred to me that there might be more direct descendants of the `<body>` here that we're not hiding, and there are indeed a few like `moz-consent-banner` and `c-banner`. It feels like a losing battle a bit to keep adding more, so here's an approach that might work out OK?

```scss
// hide sub nav and other page content when nav menu is open
body:has(.c-navigation-items.mzp-is-open) {
    & > * {
        display: none !important; /* stylelint-disable-line declaration-no-important */

        @media #{$mq-md} {
            display: block;
        }
    }

    .c-navigation {
        display: flex !important; /* stylelint-disable-line declaration-no-important */
    }
}
```

This should hide all direct descendants of `<body>` withe the exception of the navigation itself? The nice thing here would be that it also means we no longer need to keep any bedrock-specific selectors in here, making it easier to apply once it's in Protocol. Thoughts?

Reply:
Oh no wait, this isn't going to work out when resizing the browser. Hmmm :/

Reply:
OK, maybe something like this is what we need?

```scss
body:has(.c-navigation-items.mzp-is-open) {
    & > .c-sub-navigation,
    &> .moz-consent-banner.is-visible,
    &> .c-banner,
    #outer-wrapper {
        display: none !important; /* stylelint-disable-line declaration-no-important */

        @media #{$mq-md} {
            display: block !important; /* stylelint-disable-line declaration-no-important */
        }
    }
}
```

Unfortunately because the mobile menu is designed to take over the full screen, I can't see a way around this that would not require JS instead.